changelog/_unreleased/2025-04-28-inclue-seourls-for-suggestsearch-and-search-entities.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Inclue seoUrls for suggestSearch and search entities
+issue: 489
```

Comment:
this is in correct, the issue number is from different repo. IMO you can either remove it or using `shopware/SwagCommercial#489`